Neos.Neos/Classes/Fusion/ImageUriImplementation.php

```diff
@@ -143,7 +143,7 @@ public function evaluate()
         if (!$asset instanceof AssetInterface) {
             throw new \Exception('No asset given for rendering.', 1415184217);
         }
-        if ($preset !== null) {
+        if ($preset !== null && $preset !== false) {
```

Comment:
I will merge it like this if you don't come around changing it, but we could directly go and say `!empty($preset)` so also empty string is covered...